examples/simple_example.rs

```diff
@@ -0,0 +1,136 @@
+//! Centralized Telescope example with BLS signatures
+
+use alba::centralized_telescope::params::Params;
+use alba::centralized_telescope::proof::Proof;
+use alba::centralized_telescope::CentralizedTelescope;
+use blst::min_sig::{AggregatePublicKey, AggregateSignature, PublicKey, SecretKey, Signature};
+use blst::BLST_ERROR;
+use rand_chacha::ChaCha20Rng;
+use rand_core::{CryptoRng, RngCore, SeedableRng};
+use std::collections::HashMap;
+
+const DATA_LENGTH: usize = 48;
+pub(crate) type Element = [u8; DATA_LENGTH];
+
+struct AlbaSigner {
+    signing_key: SecretKey,
+    verification_key: PublicKey,
+}
+
+struct ThresholdSignature {
+    proof: Proof,
+    key_list: Vec<PublicKey>,
+}
+
+impl AlbaSigner {
+    fn new(rng: &mut (impl RngCore + CryptoRng)) -> Self {
+        let mut ikm = [0u8; 32];
+        rng.fill_bytes(&mut ikm);
+        let sk = SecretKey::key_gen(&ikm, &[])
+            .expect("Error occurs when the length of ikm < 32. This will not happen here.");
+        let pk: PublicKey = sk.sk_to_pk();
+        Self {
+            signing_key: sk,
+            verification_key: pk,
+        }
+    }
+
+    fn sign<const N: usize>(&self, msg: &[u8]) -> [u8; N] {
+        let mut signature_to_byte = [0u8; N];
+        signature_to_byte.copy_from_slice(&self.signing_key.sign(msg, &[], &[]).to_bytes());
+        signature_to_byte
+    }
+}
+
+impl ThresholdSignature {
+    fn aggregate<const N: usize>(
```

Comment:
Suggest replacing the hash maps with a slice of pairs (index, signature) like in my document. I think this would make the interface and implementation simpler. Particularly, there is no need for public keys since you can just attach the indices to complement the alba proof. So, in the language of my document, the signatures would be the data elements and the party indices would be the auxiliary data. Other options are:
1. data element (index, signature), no aux data
2. data element index, aux data signature

Reply:
Whichever structure (hashmaps, vectors, a vector of tuples...) you want to use is fine to me.
I would not recommend using indices in the proof however, as we want this to be the most straightforward example.